### PR TITLE
fix url for mainnet farm management

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

File: jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js [hunk 1]
```diff
@@ -22,12 +22,16 @@ export default {
   },
   actions: {
     getTfgridUrl: async context => {
-      var response = await tfService.getExplorer();
-      var url = JSON.parse(response.data).data.url
-      if (!url.startsWith('http')) {
-        url = `https://${url}/api/v1`;
```
Comment (m-motawea) [inline jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js:28]: you can just use this endpoint to get the explorer url `/admin/actors/identity/get_explorer_url` on development branch
result: 
```
"{\"url\": \"https://explorer.testnet.grid.tf/api/v1\"}"
```

Comment (DylanVerstraete) [reply jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js:28]: @m-motawea this is already happening right now: 
https://github.com/threefoldtech/js-sdk/blob/42e1e1c93b0b3b9992c4b4cb746689abb2cb6b64/jumpscale/packages/admin/actors/admin.py#L34
https://github.com/threefoldtech/js-sdk/blob/42e1e1c93b0b3b9992c4b4cb746689abb2cb6b64/jumpscale/packages/farmmanagement/frontend/services/farmmanagementServices.js#L5

Comment (DylanVerstraete) [reply jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js:28]: This just returns `explorer.tesnet.grid.tf` or `explorer.grid.tf` it does not return what comes after that eg: `/api/v1` in the case of testnet and `/explorer` in the case of mainnet....

Comment (m-motawea) [reply jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js:28]: sorry seems I commented on a wrong line. you don't need to hardcode "/explorer" or "/api/v1". the above actor returns the correct one as you can see in the response

Comment (DylanVerstraete) [reply jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js:28]: @m-motawea no it does not. it return : `"{\"data\": {\"type\": \"testnet\", \"url\": \"explorer.testnet.grid.tf\"}}"`

Comment (m-motawea) [reply jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js:28]: image

this is the latest on development.

**Pull request closed without merging**